api/src/main/java/com/pip/chatbot/controller/FoodController.java

```diff
@@ -28,24 +28,24 @@
                 .body(foodService.getCities());
     }
 
-    @GetMapping(value = "/city/{cityId}")
+    @GetMapping(value = "/city/{cityId}/cuisine")
     public ResponseEntity<List<Cuisine>> getCuisineForCity(@PathVariable Integer cityId) {
         return ResponseEntity
                 .ok()
                 .body(foodService.getCuisineForCity(cityId));
     }
 
-    @GetMapping(value = "/city/{cityId}/cuisine/{cuisine}")
+    @GetMapping(value = "/city/{cityId}/cuisine/{cuisine}/restaurant")
     public ResponseEntity<Restaurant> getRestaurantForCusine(@PathVariable Integer cityId, @PathVariable String cuisine) {
```

Comment:
Nie zauważliśmy literówki. W controllerze, serwisie i repo jest Cusine zamiast Cuisine. Poprawmy jeszcze to i mergujemy:)